doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,30 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+.. _iid_cv
+
+Cross-validation iterators for i.i.d. data.
+===========================================
+
+Data is Independent Identically Distributed (i.i.d.) if the individual samples
+of the data are independent of each other and are drawn from an identical
+distribution.
+
+Simply put, samples are assumed to be created from the same generative process
+and that generative process is assumed to have no memory of past generated
+samples.
+
+The following cross-validators can be used in such cases.
+
+**NOTE**
+
+However this hypothesis rarely strictly holds in practice. If one knows that
+the samples have been generated using a time-dependent process, it's safer to
+use a time-series aware cross-validation scheme. Refer :ref:`timeseries_cv`.
+Similarly if we know that the generative process has a group structure
+(samples from collected from different subjects, experiments, measurement
+devices) it safer to use group-wise cross-validation. :ref:`group_cv`.
```

Comment:
Either add ``, see :ref: ...`` or make the ``group-wise cross-validation`` the link title.